doc/storage/appendix/sra.rst

```diff
@@ -0,0 +1,919 @@
+      
+      For `DM.AUTHORISED` while a man-in the middle can eavesdrop on data in transit, all commands need to be authenticated, and all replies include some verification. By definition, the Storage Medium verifies all commands. Therefore, provided the Storage Service verifies all replies, including checking the status codes returned after it submits a write command, before releasing any data to their caller, the man-in-the-middle cannot change the command or reply without being detected.
+      
+      For `DM.SECURE_CHANNEL` if the secure channel is set up without authorisation, it would be vulnerable to a man-in-the-middle attack. Given that in this implementation we are not expecting encryption of the data - other than by the channel itself - the attacker would have full access to stored data. 
+
+   .. mitigations:: 
+      `M.Encrypt` Encryption ensures that the Man-in-the-middle does not know what data is being transferred. It also means they cannot force a specific value to be stored.  
+    
+      :mitigation:`MAC` In `DM.EXPOSED`, applying a Message Authentication Code or a signature or using an authenticated encryption scheme means that the Storage Service can check the integrity of the data when it is read back from the Storage Medium.
+      
+      :mitigation:`UniqueKeys` For `DM.AUTHORISED` and `DM.SECURE_CHANNEL`, requiring keys used by the storage partition to be unique to each instance ensures that an attacker can only learn the key used on this instance on this specific instance. They cannot construct a class break by discovering the key for a single instance. 
+   
+      :mitigation:`VerifyReplies` In `DM.AUTHORISED`, commands and replies are authenticated by the Storage Medium. Therefore, the man-in-the-middle should not be able to create a valid reply indicating that the data has been stored when it has not. Provided the Storage Service validates replies, it can be sure that the data it sent was correctly stored, and the data retrieved is the value previously stored.  
+
+      :mitigation:`AuthenticateEndpoints` In `DM.SECURE_CHANNEL`, provided the secure channel set up includes mutual authentication of the Storage Service and the Storage Medium, both sides can be sure there is no MITM. This could be because the channel uses a single key known only to both parties.  
+      
+      :mitigation:`Replay Protection`  in both `DM.AUTHORISED` and `DM.SECURE_CHANNEL` if the communication protocol includes protection against replay, normally achieved by including a nonce in the construction, it will detect attempts to replay previous commands and reject them. 

```

Comment:
Please can the mitigation tag/name be consistent in typesetting across all of the mitigations. E.g. Use CamelCase everywhere?